### PR TITLE
Add status option to pyduro

### PR DESCRIPTION
**Pull request by brjhaverkamp**
This uses the function_id 11, to get all the status parameters. This could already be sort-of gotten with raw.run, but this adds the parameter name to each of the values.

### CODE REVIEW

**Comment by brjhaverkamp** [comment]
Hi Clement, 
I added an option to get the status of the stove.
I noticed the android app does this regularly to update it's parameters.
In my plugin, I would like to use the same functionality to show the temp and sync with the status of the stove.
Hope you like it and can add it.

Bear with me, this is my first commit and pull request, so I could have screwed up completely, without knowing:-)
I got the information on the list of parameters from https://github.com/darboo44/eedomus-Aduro/blob/master/AduroApp.php
Not sure if and how to acknowledge that.

Regards,

Bert

File: src/pyduro/actions/__init__.py [hunk 1]
```diff
@@ -47,3 +47,122 @@
...
+    'drift.wifi_load',
+    'drift.co',
+    'setup.min_beholdning',
+    'compressor_countdown',
+    'drift.vacuum_aktiv',
+    'vacuum_time',
+    'drift.askeskuffekontakt',
+    'drift.askeskuffe_minutter',
+    'boiler.timer']
+)
```
Comment (clementprevot) [inline src/pyduro/actions/__init__.py]: Could you please add an empty line at the end of tis file 🙏🏻 

```suggestion
)

```

**Review by clementprevot — CHANGES_REQUESTED** [review]
This is awesome, thank you, just a minor thing to say

**Comment by clementprevot** [comment]
Hi @brjhaverkamp,

Thanks a lot for your pull request. This is a great feature!

Sorry for taking this long to answer you, but for an unknown reason, I didn't receive the notification for this pull request 😅
I've added a little thing to fix, don't hesitate to use the suggestion I made to directly fix it from the GitHub interface.

I will merge this as soon as it's OK and release a new version of the package

**Review by clementprevot — APPROVED** [review]

**Comment by clementprevot** [comment]
@brjhaverkamp the new 3.2.1 version is live. I've tweaked a bit your modification to add JSON support to the output.
And I also updated the documentation.

Hope you'll like it

**Comment by brjhaverkamp** [comment]
Hi Clement, 

Thanks for picking up the addition. Yes, I am quite happy :-) The JSON support was a good idea!
I'll update asap.

**Pull request merged**